bigchaindb/common/schema/__init__.py

```diff
@@ -0,0 +1,23 @@
+from collections import OrderedDict
+import os.path
+
+import yaml
+
+
+def ordered_load(stream):
```

Comment:
Given that the `OrderedDict` is only used for rendering the .rst, I'd move it to `generate_schema_documentation.py`.